connector/src/main/java/org/geysermc/connector/network/translators/java/world/JavaChunkDataTranslator.java

```diff
@@ -18,6 +18,7 @@
 
     @Override
     public void translate(ServerChunkDataPacket packet, GeyserSession session) {
+        session.getChunkCache().addToCache(packet.getColumn());
```

Comment:
Hold off on working with the chunk cache for now (mainly pertaining to block collisions). I'd like to discuss this a bit more in detail on Discord and maybe a better way to approach this since chunk caching can eat up RAM very fast. Chunk caching/block collisions should remain as a separate PR anyway.